### DISCUSSION TITLE
！请问会适配valine评论插件吗

### DISCUSSION THREAD

**Discussion by New-ling** [MaLuns/hexo-theme-async#60] [Q&A 疑难杂症]
……我是一点都看不懂鸭
https://valine.js.org/quickstart.html#%E9%85%8D%E7%BD%AE
👆关于官方文档的【配置】这一项，我想知道放在哪里……

**Comment by MaLuns** [comment]
暂时不会适配这个😀，如果你想使用这个评论插件 ，[可以参考文档集成其他评论插件说明](https://hexo-theme-async.imalun.com/guide/third-party-support.html#%E9%9B%86%E6%88%90%E5%85%B6%E4%BB%96%E8%AF%84%E8%AE%BA%E6%8F%92%E4%BB%B6)
